### PR TITLE
Two Mongo aggregations

### PR DESCRIPTION
**Pull request by SabreCat**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

Creates a new folder in the repo, `/reports`, to house scripts useful for pulling various data from a live system.

Two examples are included:
* `guild-count.js`: Counts how many Guilds each user is a member of, printing the numbers of users that belong to each number of Guilds.
* `sound-themes.js`: Counts how many users employ each of the available sound effect themes.

Queries limit the users polled to those active in the 2017 calendar year. The query can be adjusted to pull data for different populations, as desired.

[//]: # (Put User ID in here - found in Settings -> API)

### CODE REVIEW

File: reports/guild-count.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// How many Guilds are users members of? Outputs counts of users with each number of Guilds
+
+var aggregationCursor = db.users.aggregate(
```
Comment (paglias) [inline reports/guild-count.js]: we can use most es6 features with Node 6, can this use `let`?

Comment (TheHollidayInn) [reply reports/guild-count.js]: Maybe we should just add this directory to linting? Since I think we will be adding on to this.

**Comment by paglias** [comment]
Yeah good idea

Il 18 Apr 2017 7:44 PM, "Keith Holliday" <notifications@github.com> ha
scritto:

File: reports/guild-count.js (hunk 1 above)
Comment (SabreCat) [reply reports/guild-count.js]: Yeah, the lint tests are actually failing, I think it was included by default >.< I'll see if it'll run after cleaning up the linting.

**Comment by SabreCat** [comment]
Gonna redo this probably. Closing for now.

**Pull request closed without merging**